tests/cli-test.js

```diff
@@ -110,5 +117,18 @@ QUnit.module('codemod-cli', function(hooks) {
         })
       );
     });
+
+    QUnit.module('test', function() {
+      QUnit.test(
+        'should pass for a basic project',
+        wrap(function*(assert) {
+          yield execa(EXECUTABLE_PATH, ['generate', 'codemod', 'main']);
+          yield execa(EXECUTABLE_PATH, ['generate', 'fixture', 'main', 'this-dot-owner']);
+
+          let result = yield execa(EXECUTABLE_PATH, ['test']);
+          assert.equal(result.code, 0, 'exited with zero');
```

Comment:
So, we only care that it doesn’t error by default?

Reply:
Ya, at this point in the test we have a codemod (that does nothing) and two empty fixtures. Since the input and output are both `''` we just test that it passes...